crates/cairo-lint-core/src/lints/mod.rs

```diff
@@ -5,4 +5,5 @@ pub mod double_parens;
 pub mod duplicate_underscore_args;
 pub mod ifs;
 pub mod loops;
+pub mod manual_ok_or;
```

Comment:
create a `manual` module so all the manual lints will go there. you can check the `ifs` module if you need an example